PROJECT

```diff
@@ -1,51 +1,132 @@
 domain: 3scale.net
-layout: go.kubebuilder.io/v2
+layout: go.kubebuilder.io/v3
 multigroup: true
 projectName: 3scale-operator
 repo: github.com/3scale/3scale-operator
 resources:
-- group: apps
+-
+  # TODO(user): Uncomment the below line if this resource implements a controller, else delete it.
+  # controller: true
+  group: apps
   kind: APIManager
+  # TODO(user): Update the package path for your API if the below value is incorrect.
+  path: k8s.io/api/apps/v1alpha1
   version: v1alpha1
-- group: apps
+-
+  # TODO(user): Uncomment the below line if this resource implements a controller, else delete it.
+  # controller: true
```

Comment:
This too? Can you review this file again please

Reply:
done. Thank you